packages/bitcoin/redux/reducers.js

```diff
@@ -253,7 +274,7 @@ const bitcoinTxUpdateToken = (state, { token, address, parentAddress }) => {
   }
 }
 
-const bitcoinTxUpdateAmount= (state, { amount, address, parentAddress }) => {
+const bitcoinTxUpdateAmount = (state, { amount, address, parentAddress }) => {
```

Comment:
A transaction's amount got once form the middleware and never updates.
Incorrect name?